pkg/cmd/install.go

```diff
@@ -130,7 +131,7 @@ func expandPath(path string) string {
 	return res
 }
 
-func SSHAgent() ssh.AuthMethod {
+func sshAgent(publicKeyPath string) ssh.AuthMethod {
 	if sshAgentConn, err := net.Dial("unix", os.Getenv("SSH_AUTH_SOCK")); err == nil {
```

Comment:
What happens on Windows for this?

Reply:
That should work if you're in cygwin or wsl.
Otherwise it will fail and the user will be asked to enter his passphrase.

Reply:
👍 

I do prefer using the "standard" Go packages which you have here if you think that it's workable to keep with them.